### ISSUE TITLE
Contrasting concepts (antonyms)

### ISSUE THREAD

**Issue by Kristina-Pianykh** [concepticon/concepticon-data#938]
The study is available here: https://portal.research.lu.se/ws/files/5315771/1504520.pdf. The Appendix on pages 173-177 includes all the studies words in each specific semantic field. It is not clear to me yet if all these words should be added to Concepticon and when so, how to do it in the Concepticon-friendly way.

**Comment by LinguList** [comment]
Yes, I think it is better to see if there's a list with translations and the most prominent ones. There are some adjectives so spepcial that it isn't worth adding them...
